apcd_cms/src/apps/common_api/views.py

```diff
@@ -5,13 +5,16 @@
 from apps.base.base import BaseAPIView, APCDAdminAccessAPIMixin, APCDGroupAccessAPIMixin
 from datetime import datetime
 import logging
+import json
 
 logger = logging.getLogger(__name__)
 
 
 class EntitiesView(APCDGroupAccessAPIMixin, BaseAPIView):
     def get(self, request, *args, **kwargs):
-        submitters = apcd_database.get_submitter_info(request.user.username)
+        from_admin = json.loads(request.GET.get('from_admin', False))  # for admin submissions page, need all submitters, not just for requesting user
```

Comment:
To prevent non admin users from request data just by using query param.
```suggestion
        from_admin = json.loads(request.GET.get('from_admin', False))  # for admin submissions page, need all submitters, not just for requesting user
        if from_admin and not is_apcd_admin(request.user):
            return JsonResponse({'error': 'Unauthorized'}, status=403)
```